traveler-ios-ui/TravelerKitUI/ViewControllers/CatalogItemLoadingViewController.swift

```diff
@@ -9,13 +9,7 @@
 import UIKit
 
 class CatalogItemLoadingViewController: UIViewController {
```

Comment:
Same here.....

But also..don't we want the specific catalog looking ones ever?

Reply:
Yeah I just used the generic one for speed purposes. Maybe we can add a low priority ticket to improve loading pages in flows?.